CHANGELOG.md

```diff
@@ -10,7 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 
 ### Changed
-
+- Changed `CloudForCustomers` methods: `_to_records_report` and `_to_records_other` to handle empty response in proper way
```

Comment:
What is the 'proper way'? Please specify what was changed.

Reply:
Changed